src/pages/juniorPromise/components/BottomSheetBg.tsx

```diff
@@ -0,0 +1,117 @@
+import { ReloadIc } from '@assets/svgs';
+import styled from '@emotion/styled';
+interface BottomSheetPropType {
+  isSheetOpen: boolean;
+  handleSheetOpen: () => void;
+  handleSheetClose: () => void;
+}
+
+export const BottomSheet = ({ isSheetOpen, handleSheetClose }: BottomSheetPropType) => {
+  return (
+    <>
+      <Background $isSheetOpen={isSheetOpen} onClick={handleSheetClose} />
+      <BottomSheetWrapper $isSheetOpen={isSheetOpen}>
+        <TitleLayout>
+          <Line />
+          <Title>원하는 선배를 찾아볼까요?</Title>
+          <Desc>계열, 직무로 원하는 선배를 찾을 수 있어요.</Desc>
+        </TitleLayout>
+        <ButtonLayout>
+          <ReloadIcon type="reset">
+            <ReloadIc />
+          </ReloadIcon>
+          <ExitBottomSheet type="button" onClick={handleSheetClose}>
+            적용할래요
+          </ExitBottomSheet>
+        </ButtonLayout>
+      </BottomSheetWrapper>
+    </>
+  );
+};
+// const TestBtn = styled.button`
+//   display: flex;
+// `;
+
+const Background = styled.div<{ $isSheetOpen: boolean }>`
+  display: ${({ $isSheetOpen }) => ($isSheetOpen ? 'flex' : 'none')};
+  position: fixed;
+  top: 0;
+  z-index: 2;
+
+  width: 100%;
+  height: 100dvh;
+
+  background: ${({ theme }) => theme.colors.transparentBlack_65};
+`;
+
+const BottomSheetWrapper = styled.form<{ $isSheetOpen: boolean }>`
+  display: flex;
+  flex-direction: column;
+  position: fixed;
+  bottom: 0;
+  z-index: 4;
+
+  width: 100%;
+  height: auto;
+  border-radius: 16px 16px 0 0;
+
+  background: ${({ theme }) => theme.colors.grayScaleWhite};
+
+  opacity: ${({ $isSheetOpen }) => ($isSheetOpen ? 1 : 0)};
+  transform: translateY(${({ $isSheetOpen }) => ($isSheetOpen ? '0' : '100%')});
+
+  transition:
+    transform 250ms ease-out,
+    opacity 250ms ease-out;
```

Comment:
P3) transition 속성에서 ease-in-out 값을 사용하면 애니메이션의 시작과 끝 부분이 더 부드럽게 보이도록 만들어줍니다!! ease-out 대신 사용해보는게 어떨까요??
```suggestion
  transition:
    transform 250ms ease-in-out,
    opacity 250ms ease-in-out;
```

Reply:
오! 훨씬 보드라워요!!! 수정완료했습니다!! 감사합니다!!